letsencrypt-nginx/letsencrypt_nginx/tests/util.py

```diff
@@ -60,6 +61,13 @@ def get_nginx_configurator(
         name="nginx",
         version=version)
     config.prepare()
+    # also make a general client config for good measure...
+    namespace = mock.MagicMock(
```

Comment:
This namespace object should probably be the config created inline on line 51.  

You avoid differing config objects, which would create some confusion.  Given the testing suite already relies on some of these config variables.  (The config_dir parameter of this function is an existing directory and the tests rely on writing to it.)